docs/overview.md

```diff
@@ -0,0 +1,149 @@
+# Cluster Provisioning with CAPI
+
+This guide covers how to deploy a Canonical Kubernetes multi-node cluster using Cluster API (CAPI).
+
+## Install `clusterctl`
+
+The `clusterctl` CLI tool manages the lifecycle of a Cluster API management cluster. To install it, follow the [upstream instructions]. Typically, this involves fetching the executable that matches your hardware architecture and placing it in your PATH. For example, at the time this guide was written, for `amd64` you would:
+
+```sh
+curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.7.3/clusterctl-linux-amd64 -o clusterctl
+sudo install -o root -g root -m 0755 clusterctl /usr/local/bin/clusterctl
+```
+
+### Set up a management Cluster
```

Comment:
Maybe we can put the management cluster setup in a general doc and do something like this:
```
## Prerequisites

To follow this guide, you will need:

- A CAPI management cluster
- ...

Please refer to the [Something Guide](some-doc.md) for instructions.
```
Asking because my docs are going to need this too :D